main.go

```diff
@@ -42,30 +50,8 @@ func main() {
 		cancelApp()
 	}()
 
-	// Load environment variables from a .env file.
-	if err := config.LoadEnv(); err != nil {
-		logger.Error("Failed to load environment variables", "error",
-			err)
-		os.Exit(1)
-	}
-
-	// Load configuration settings from environment variables.
-	cfg, err := config.LoadConfig()
-	if err != nil {
-		logger.Error("Failed to load configuration", "error", err)
-		os.Exit(1)
-	}
-
-	// Parse the fuzzing cycle duration from configuration (e.g., "20s").
-	cycleDuration, err := time.ParseDuration(cfg.FuzzTime)
-	if err != nil {
-		logger.Error("Error parsing cycle duration", "durationString",
-			cfg.FuzzTime, "error", err)
-		os.Exit(1)
-	}
-
 	// Start the continuous fuzzing cycles.
-	scheduler.RunFuzzingCycles(appCtx, logger, cfg, cycleDuration)
+	scheduler.RunFuzzingCycles(appCtx, logger, cfg, cfg.FuzzTime)
```

Comment:
Since we're already passing `cfg`, we shouldn't need to pass `cfg.FuzzTime` anymore.